Use Octal Numbered Label Names for VS Channel Paths

jkreienh opened the pull request:
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-ballard-arinc429-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

1. An Octal numbered label name now appears in the System Explorer and VeriStand channel path if the `<labelOctal>` tag is specified in the parameter configuration file. 
2. The correct Octal and Decimal representation is displayed on the System Explorer label page regardless of the `<labelOctal>` or `<labelDecimal>` tag selection.
3. Called `NI VeriStand - Visually Update the System Explorer Tree.vi` to refresh the System Explorer tree after import.

### Why should this Pull Request be merged?

1. Arinc-429 users typically refer to labels in an Octal Number format and would expect VeriStand channel paths to reflect this representation for reference and mapping when the `<labelOctal>` tag is specified in the parameter configuration file. Fixes #211 
2. The System Explorer tree should refresh after import. Fixes #249 

### What testing has been done?

1. Successfully completed all VI Tester steps. 
4. Configured and imported a loopback test that defines Label 7 and 350 as octal using the `<labelOctal>` tag.
5. Observed the correct values for labels defined as octal and decimal in the label page.
6. Deployed and observed the correct behavior when updating channel values.
7. Configured and imported a loopback test that defines Label 7 and 232 as decimal using the `<labelDecimal>` tag.
8. Observed the correct values for labels defined as octal and decimal in the label page.
9. Deployed and observed the correct behavior when updating channel values.
10. Observed tree refresh after import.

buckd commented:
/AzurePipelines run

buckd reviewed (commented):
**Label (Dec or Oct)** is fairly ambiguous since you don't know which it is. Because it's only used for constructing the display/channel names in the system definition, I would prefer that item be called something like **Label (Display)**. This way I know what it's used for when reading the code and removes the confusion of trying to figure out if it's decimal or octal because that knowledge isn't necessary.

jkreienh commented:
@buckd - I thought this might be confusing.  I've updated from **Label(Dec or Oct)** to **Label(Display)**

### What testing has been done?

1. Successfully completed all VI Tester steps.
2. Deleted all instances of the Custom Device 
3. Configured and imported a loopback test that defines Label 7 and 350 as octal using the `<labelOctal>` tag.
4. Observed the correct values for labels defined as octal and decimal in the label page.
5. Deployed and observed the correct behavior when updating channel values.

buckd commented:
/AzurePipelines run

buckd commented on Source/Custom Device Support/Ballard ARINC 429 Support.lvproj:
This looks like only dependencies were changed and nothing of substance in the `.lvproj` changed. Revert this file.

buckd reviewed (commented):
Everything looks good, except let's revert the project file since there are no actual changes.

buckd commented:
/AzurePipelines run
